### DISCUSSION TITLE
Allow CSV Uploads for DOI Registration in Fabrica

### DISCUSSION THREAD

**Discussion by adambuttrick** [datacite/datacite-suggestions#109] [General Suggestions]
### What is the problem that your suggestion solves?

Non-technical users need the ability to register and update records in bulk without generating XML or using the API.  Many organizations manage large numbers of objects that require DOI assignment but lack the means or capacity to register them in this fashion, leaving tedious, repetitive form registration as their only option. 

### What solution might meet your needs?

Fabrica should allow for DOI registration and updates by CSV upload.

### Your name

Adam Buttrick

### Your organization

California Digital Library

### Is there anything else you would like to share?

ROR's curation pipeline manages record creation and updates with CSVs and is a potential example for how to implement. 

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by codycooperross** [comment]
Thank you, Adam! This helps us better understand the needs around bulk update and create of DOI metadata. We'll watch this thread for further discussion, so feel free to provide additional context and related suggestions here. 

When thinking about the CSV format, can you describe how a non-technical user might want to contribute nested and repeating data structures like creators with nameIdentifiers and affiliations, for example? Is this something you handle in the ROR curation pipeline?
